components/CancelModal.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, MouseEventHandler } from 'react'
+import ActionButton from './ActionButton'
+import { ActionButtonProps } from './ActionButton'
+import { FocusOn } from 'react-focus-on'
+
+export interface CancelModalProps {
+  buttonText: string
+  description: string
+  isOpen: boolean
+  onClick: MouseEventHandler<HTMLButtonElement>
+  buttons: ActionButtonProps[]
+}
+
+const CancelModal: FC<CancelModalProps> = ({
+  buttonText,
+  description,
+  buttons,
+  onClick,
+  isOpen,
+}) => {
+  return (
+    <>
+      <ActionButton text={buttonText} onClick={onClick} style="primary" />
```

Comment:
```suggestion
      <ActionButton text={buttonText} onClick={onClick} />
```
The style needs to be "default", we can't have two primary buttons on the same page.
You could consider passing the style to this component... but still needs to be set to default.
_FYI, this rule doesn't apply to the buttons in the modal, as we typical avoid showing bias towards an option_

Reply:
Good catch. We'll need to update the default styling to be in line with the Canada.ca design pattern in a separate PR. It's mostly fine, just need to update the colors slightly.